lib/post_creator.rb

```diff
@@ -431,7 +431,7 @@ def track_latest_on_category
 
   def ensure_in_allowed_users
     return unless @topic.private_message? && @topic.id
-    return if @post.whisper?
+    return if @post.whisper? || @post.post_type == Post.types[:small_action]
```

Comment:
I wonder if it makes sense to have a `@post.small_action?` helper?

Reply:
I was surprised to see one does not exist, but I don't think it is worth it. We only check for small actions in 3 places. In contrast, the whisper helper is used much more.

Reply:
Fair enough 👍